fv3core/fv3core/stencils/fv_subgridz.py

```diff
@@ -838,15 +843,19 @@ def make_storage():
         self._tmp_cpm = make_storage()
         self._ratios = {0: 0.25, 1: 0.5, 2: 0.999}
 
-    def __call__(self, state: Mapping[str, pace.util.Quantity], timestep: float):
+    def __call__(
+        self, state: DycoreState, u_dt: Quantity, v_dt: Quantity, timestep: float
```

Comment:
Ah OK this makes sense. Can keep this as-is then. If we move fv_subgridz, we can do one of those two things - this can get merged into PhysicsState in some way, or at least defined in `pace.physics`.